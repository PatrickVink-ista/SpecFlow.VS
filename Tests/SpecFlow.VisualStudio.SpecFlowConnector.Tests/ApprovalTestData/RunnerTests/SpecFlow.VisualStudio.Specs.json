{
  "TargetFolder": "..\\..\\..\\..\\SpecFlow.VisualStudio.Specs\\bin\\Debug\\net472",
  "TestAssembly": "SpecFlow.VisualStudio.Specs.dll",
  "ConfigFile": null
}
