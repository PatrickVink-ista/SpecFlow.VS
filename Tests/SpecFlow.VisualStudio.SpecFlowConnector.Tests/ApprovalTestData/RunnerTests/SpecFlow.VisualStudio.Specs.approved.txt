{
  "stepDefinitions": [],
  "sourceFiles": {},
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "TechTalk.SpecFlow.dll",
    "SFFileVersion": "3.9.40.46609",
    "SFProductVersion": "3.9.40+11b6bbec89",
    "SourcePaths": "2",
    "StepDefinitions": "82",
    "TargetFramework": ".NETFramework,Version=v4.7.2",
    "TypeNames": "7"
  }
}